sklearn/linear_model/_ransac.py

```diff
@@ -478,7 +478,7 @@ def predict(self, X):
             Returns predicted values.
         """
         check_is_fitted(self)
-
+        X = self._validate_data(X, accept_sparse='csr', reset=False)
```

Comment:
Another approach is to not validate if `self.estimator_.n_features_in_` is defined and delgate it to `self.estimator_`.

Reply:
Indeed, think I am in favor to delegating the check to the underlying estimator.

Reply:
If we are delegating, I think it would good to adjust the error message: https://github.com/scikit-learn/scikit-learn/pull/18585 to not include the estimator name.